docs/using.rst

```diff
@@ -296,7 +297,21 @@ need to issue this command in normal circumstances.
 
 ``--expand`` tells Certbot to update an existing certificate with a new
 certificate that contains all of the old domains and one or more additional
-new domains.
+new domains. With the ``--expand`` option, use the ``-d`` option to specify
+one or more domains.
+
+Example:
+
+.. code-block:: none
+
+  certbot --expand --cert-name existing.com -d example.com,newdomain.com
+
+If you prefer, you can specify the domains individually like this:
+
+.. code-block:: none
+
+  certbot --expand --cert-name existing.com -d example.com -d newdomain.com
+
```

Comment:
This doesn't seem to be an accurate representation of what @bmw said in #4660 
I think this section should be elaborated to either specify that expand and cert-name are different commands, or that cert-name should always be used instead of expand.
I think I prefer the latter, since --expand can lead to an unpredicted cert getting modified, where as cert-name leads to more control.